packages/gasket-plugin-intl/test/build-manifest.test.js

```diff
@@ -0,0 +1,78 @@
+const assume = require('assume');
+const sinon = require('sinon');
+const proxyquire = require('proxyquire');
+const path = require('path');
+
+const writeFileStub = sinon.stub().callsFake((...args) => {
+  args[args.length - 1](null, true);
+});
+
+const buildManifest = proxyquire('../lib/build-manifest', {
+  fs: {
+    writeFile: writeFileStub
+  }
+});
```

Comment:
IMHO, for better test independence, move these into the `beforeEach` so you get a brand new instance each time.